src/main/java/org/sonar/plugins/scm/perforce/PerforceConfiguration.java

```diff
@@ -90,6 +91,15 @@ public PerforceConfiguration(Settings settings) {
         .subCategory(CATEGORY_PERFORCE)
         .index(3)
         .build(),
+      PropertyDefinition.builder(CLIENT_IMPERSONATED_HOST_PROP_KEY)
+        .name("Client impersonated hostname")
+        .description("Name of the host computer to impersonate, per the <a href=\"https://www.perforce.com/perforce/r15.1/manuals/cmdref/P4HOST.html\">Perforce documentation</a>")
```

Comment:
Do you confirm you have checked that HTML is allowed (and actually work) in property description?